src/Lykke.Job.BlockchainCashoutProcessor/Modules/CqrsModule.cs

```diff
@@ -20,7 +20,8 @@ namespace Lykke.Job.BlockchainCashoutProcessor.Modules
 {
     public class CqrsModule : Module
     {
-        private static readonly string Self = BlockchainCashoutProcessorBoundedContext.Name;
+        public static readonly string Self = BlockchainCashoutProcessorBoundedContext.Name;
+        public static readonly string CrossClientContext = $"{BlockchainCashoutProcessorBoundedContext.Name}.cross-client";
```

Comment:
Let's do it as single context, but two sagas in it. Just name cross client saga as $"{Self}.cross-client-saga"